lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -67,6 +70,7 @@
 @import '../../../../styles/settings/media-queries';
 @import '../../../../styles/settings/radiuses';
 @import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/shadows';
 @import './border-colors';
 
 $rich-list-item-icon-colors: (
```

Comment:
Look at the https://github.com/bethinkpl/design-system/blob/master/lib/styles/settings/_icons.scss#L17 
@fraszczakszymon already solved a problem with the icon color list. Look if you can reuse the solution, please :)